gbpservice/nfp/orchestrator/coal/networking/nfp_gbp_network_driver.py

```diff
@@ -42,6 +41,13 @@ def update_port(self, token, port_id, port):
                                                        port)
         return pt['port_id']
 
+    def get_neutron_port_details(self, token, port_id):
```

Comment:
RPM that is the optimization, we cannot do like that, please see "self.get_port_id" does "get_policy_target" which is an unnecessary REST API, while the caller has neutron port_id already.